src/uct/ib/ud/base/ud_iface.c

```diff
@@ -576,6 +585,13 @@ ucs_config_field_t uct_ud_iface_config_table[] = {
      "Enable checking destination GID for incoming packets of Ethernet network.\n"
      "Mismatched packets are silently dropped.",
      ucs_offsetof(uct_ud_iface_config_t, dgid_check), UCS_CONFIG_TYPE_BOOL},
+
+    {"MAX_WINDOW", UCS_PP_MAKE_STRING(UCT_UD_CA_MAX_WINDOW),
+     "Max congestion avoidance window. Should be >= "
+      UCS_PP_MAKE_STRING(UCT_UD_CA_MIN_WINDOW) " and <= "
```

Comment:
would not help formatting look weird this way?
with just a number on the new line

Reply:
could you please be more specific - what format do you propose?

Reply:
@brminich there is no "\n" in the string..

Reply:
how would it look like when you invoke ucx_info -f?
Is it like below?
```
Max congestion avoidance window. Should be >= 
8 and <=
1024

```

Reply:
@brminich there is no "\n" in the string..

Reply:
there is no new line, plz disregard

Reply:
there is no new line, plz disregard